ta4j-core/src/main/java/org/ta4j/core/indicators/EMAIndicator.java

```diff
@@ -43,4 +43,9 @@ public class EMAIndicator extends AbstractEMAIndicator {
     public EMAIndicator(Indicator<Num> indicator, int barCount) {
         super(indicator, barCount, (2.0 / (barCount + 1)));
     }
+
+    @Override
+    public int getUnstablePeriod() {
+        return 0;
```

Comment:
EMA requires more than 0 periods to be computed.

Depending on how the EMA is initialized, it might require at least the number of period.


Ex: 
For a 9 EMA, the seed might be initialized with the 9 SMA.
This means that, to compute the 9 EMA, you'll need at least 9 periods for the SMA 


This topic on reddit was very insightful !

https://www.reddit.com/r/algotrading/comments/bbnv8c/how_many_candles_minimum_to_get_an_accurate_ema/